test/acceptance/cli-test.js

```diff
@@ -145,10 +145,17 @@ describe('ember-template-lint executable', function () {
     });
 
     describe('given no path', function () {
-      it('should print errors', function () {
+      // todo doublecheck if we need this guard
+      // you don't have to specify path on Windows OS
+      // if (process.platform === 'win32') {
+      //   return;
+      // }
```

Comment:
Once we decide what we need here, we'll need to delete or restore this (commenting so we don't forget).